v5/owner.apib

```diff
@@ -0,0 +1,449 @@
+
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200 (application/json)
+    + Attributes (Parking space)
+
+### Retrieve a Parking space [GET /owner/parking-spaces/{parking_space_id}]
+Retrieves the details of one singe parking space.
+
++ Parameters
+    + parking_space_id: `005c4826-4e28-11e3-a675-d43d7eece53d` (string, required) - The id or code of the parking space
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200 (application/json)
+    + Attributes (Parking space)
+
++ Response 404
+Returned if parking space was not found or it doesn't belong to the user.
+    + Body
+
+### Update a Parking space [PUT /owner/parking-spaces/{parking_space_id}]
+Updates the parking space by setting the values of the parameters passed. Any parameter not provided will be left unchanged.
+
++ Parameters
+    + parking_space_id : `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+
++ Request  (application/json)
+    + Attributes
+        + `status`: `inactive` (string)
+        + `code`: `ASDF12` (string)
+        + `type`: `offstreet` (string)
+        + `price` (Owner Price)
+        + `options` (array) - describes which options are available at the location and can contains each of the following elements. The list can be **empty**.
+            + (enum)
+              + `excess length`
+              + `indoor`
+        + `post_booking_info`: `Hallo, bitte lassen Sie mich allein.`
+        + `earliest_start`: `-15` (number) - bookable_range_start in minutes
+        + `latest_end`: `43200` (number) - bookable_range_end in minutes (should be max 183 days)
+        + `minimum_duration`: 30 - booking_duration_min
+        + `maximum_duration`: 43200 - booking_duration_max (should be max 183 days)
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200
+    + Attributes (Parking space)
+
++ Response 404
+Returned if parking space was not found ot it doesn't belong to the user.
+    + Body
+
++ Response 422
+Returned if an error occurred.
+    + Attributes (Api Error)
+
+### List parking space descriptions [GET /owner/parking-spaces/{parking_space_id}/descriptions]
+Returns all descriptions for one parking space
+
++ Parameters
+    + parking_space_id: `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200 (application/json)
+    + Attributes (Description)
+
++ Response 404
+Returned if parking space was not found or it doesn't belong to the user.
+    + Body
+
+### Create a new Description [POST /owner/parking-spaces/{parking_space_id}/descriptions]
++ Request (application/json)
+    + Attributes
+        + `locale`: `de_DE` (string, required)
+        + `content`: `Hallo, ich bin ein Parkplatz.` (string, required)
```

Comment:
Again, use something that makes at least a little bit sense to the reader, e.g. `This is a comfortable parking space with 24 hour security service.`

Reply:
Done